projects/plugins/jetpack/changelog/fix-out-of-bounds-menu

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: bugfix
+Comment: Fix the submenus shown outside of screen
```

Comment:
It may be worth specifying that this fix is in the masterbar module.
```suggestion
Comment: Fix the submenu positioning for the masterbar left nav
```